tests/operators/test_latest_only_operator.py

```diff
@@ -91,25 +91,33 @@ def test_skipping_non_latest(self):
         downstream_task2.set_upstream(downstream_task)
         downstream_task3.set_upstream(downstream_task)
 
+        data_interval = self.dag.infer_automated_data_interval(timezone.coerce_datetime(DEFAULT_DATE))
```

Comment:
I don’t think we need to call this? Instead this can simply use a literal interval, say `(DEFAULT_DATE, DEFAULT_DATE)`.

Reply:
Sure, should I also change the others as well, like `(END_DATE, END_DATE + INTERVAL)`?

Reply:
Yeah, change those as appropriate.